libyul/AsmPrinter.h

```diff
@@ -46,12 +46,20 @@ struct Dialect;
 class AsmPrinter
 {
 public:
+	enum class Mode
+	{
+		OmitDefaultType,
+		FullTypeInfo
+	};
```

Comment:
Damn, looks like github ate my last comment. I thought I suggested renaming this enum, but the comment missing :(

I think it would be better to be more specific than `Mode`, since this is only about one, very specific aspect of printing and if we add more options, we'd need more such enums.
```suggestion
	enum class TypePrinting
	{
		OmitDefault,
		Full
	};
```

Reply:
Yeah that's fair, I'll update it tomorrow :)